src/client/keycode.h

```diff
@@ -4,62 +4,70 @@
 
 #pragma once
 
-#include "exceptions.h"
 #include "irrlichttypes.h"
+#include "cmake_config.h" // USE_SDL2
```

Comment:
not used in header

Reply:
https://github.com/luanti-org/luanti/pull/14964/commits/604faac1c3a42bc51347bfea9dcad65b7bbf09f6